code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -85,6 +85,12 @@
 	color = "#e99e12" // rgb: 233, 158, 18
 	taste_description = "apples"
 
+/datum/reagent/consumable/applejuice/on_mob_life(mob/living/carbon/user)
+	if(user.job == "Medical Doctor" || user.job == "Paramedic" || user.job == "Chief Medical Officer")
```

Comment:
paramedics aren't exactly doctors

Reply:
paramedics are just mobile doctors

Reply:
they aren't licensed doctors

Reply:
they have a doctorate in validhunting so its fine by me

Reply:
the thematic is off so it sorta defeats the purpose